forest_plot_lib.py

```
@@ -278,11 +297,15 @@ def create(
 
         y_pos = list(range(len(self.data)))
         
-        fig.add_trace(go.Scatter(x=[0]*len(y_pos), y=y_pos, text=self.data['__display_label'], mode="text", textposition="middle right", textfont=dict(size=13, color="black"), hoverinfo="none", showlegend=False), row=1, col=1)
-        fig.add_trace(go.Scatter(x=[0]*len(y_pos), y=y_pos, text=self.data['__display_est'], mode="text", textposition="middle center", textfont=dict(size=13, color="black"), hoverinfo="none", showlegend=False), row=1, col=2)
+        # Add traces with resilience (though unlikely to fail, good practice)
+        def _add_traces():
```

Ruff (0.14.10)

Missing return type annotation for private function `_add_traces`

Add return type annotation: `None`

(ANN202)